cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -276,14 +241,82 @@ public static String randomIsbn13() {
         return FAKER.code().isbn13();
     }
 
-
     public static String randomIssn() {
         String issnMissingChecksum = random7DigitNumber();
         return createValidIssn(issnMissingChecksum);
     }
 
+    public static CristinBookOrReportMetadata randomBookOrReportMetadata() {
+        return CristinBookOrReportMetadata
+            .builder()
+            .withIsbn(randomIsbn13())
+            .withPublisherName(randomString())
+            .withNumberOfPages(randomString())
+            .withSubjectField(randomSubjectField())
+            .withBookSeries(randomBookSeries())
+            .withIssue(randomString())
+            .withVolume(randomString())
+            .build();
+    }
+
     private static String random7DigitNumber() {
-        return String.valueOf((int)(Math.random() * 9000000) + 1000000);
+        return String.valueOf((int) (Math.random() * 9000000) + 1000000);
+    }
+
+    private static String createValidIssn(String issnMissingChecksum) {
```

Comment:
Have I seen this code somewhere before in a shared lib?

Reply:
in nva-publication-api. Maybe we should create a small class in testutils for creatting random data